dashboard/grid_builder.py

```diff
@@ -88,8 +96,32 @@ def has_issues(self, p: package.Package) -> bool:
                 if not package_2_self_conflict and \
                                 result['status'] != 'SUCCESS':
                     return True
+
+            # Whether the package has deprecated dependencies or not
+            if self.deprecated_deps[p.install_name][1]:
+                return True
+
         return False
 
+    def get_deprecated_deps(self):
+        """
+        Returns True if there are deprecated dependencies for a
+        given package.
+        """
+        finder = deprecated_dep_finder.DeprecatedDepFinder()
+        deprecated_deps = list(finder.get_deprecated_deps())
+
+        results = {}
+        for item in deprecated_deps:
```

Comment:
Could you use tuple unpacking so that item[0][X] has a name?

Reply:
Done.